src/raiden_libs/service_registry.py

```diff
@@ -223,6 +229,19 @@ def format_token_amount(amount: float) -> str:
     return format_token_amount
 
 
+def get_approximate_registration_cost(web3: Web3) -> int:
+    gas_costs = gas_measurements()
+    gas_cost = (
+        gas_costs["CustomToken.approve"]
+        + gas_costs["ServiceRegistry.deposit"]
+        + gas_costs["ServiceRegistry.setURL"]
+    )
+
+    gas_price = web3.eth.generate_gas_price() or 1
```

Comment:
in what scenario does this web3 method return `None`? Shouldn't the user be informed about that scenario (either warning and no gas estimation or a failure) rather than assuming the lowest possible gas_price then?
For this function this would mean returning None. Conversion to user-facing output would be done in the upper layers

Reply:
Good question, seems like this only happens when no gas price strategy is set. So I'd just remove the `None` case.